addons/civilian_statemachine/functions/fnc_getGroupVehicles.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function returns all vehicles owned by group members.
+ *
+ * Arguments:
+ * 0: Group to return it's vehicles <GROUP>
+ *
+ * Return Value:
+ * 0: Group's owned vehicles <ARRAY>
+ *
+ * Example:
+ * [group bob] call afsk_civilian_statemachine_fnc_getGroupVehicles
+ *
+ * Public: No
+ */
+
+params ["_group"];
+
+// Get all group vehicles
+private _groupVehicles = [];
+{
+    private _vehicle = _x getVariable [QGVAR(ownedVehicle), objNull];
+    if (!isNull _vehicle) then {
+        _groupVehicles pushBackUnique _vehicle;
+    };
+} forEach (units _group);
```

Comment:
Might be faster due to SimpleVM loop optimizations (build 151218+ or profiling), I did not benchmark this.
```suggestion
private _groupVehicles = units _group apply {_x getVariable [QGVAR(ownedVehicle), objNull]} select {!isNull _x};
_groupVehicles arrayIntersect _groupVehicles // return
```